lib/Server.js

```diff
@@ -270,8 +270,10 @@ class Server {
     const webpackURL = 'http://' + this.hostname + ':' + this.webpackPort;
     const hot = this.hot;
     const publicPath = hot ? (webpackURL + '/') : null;
+    const platform = this.entry.substring(this.entry.search(/\./) + 1);
```

Comment:
> Ideally, the user should be able to name their entry points however they like.

Agreed. The React Native packager decides the platform based on the `platform` URL parameter, fwiw. Would be nice if we could just keep that convention.

> We'll also probably need to have different webpack 'externals' configs for each platform.

Yeah, because the module list resolves differently.

> Hmm, I wonder if webpack allows different externals for different entry points.

Another thing to consider is the platform-based module resolution. If I do `require('./foobar')` with the React Native packager within a bundle that's been loaded with `?platform=<platform>`, it will resolve to `./foobar.<platform>.js` or `./foobar.js`. Having both `foobar.ios.js` and `foobar.android.js` being importable as `./foobar` is quite powerful. Because you can push platform-dependent differences into their own modules/React components and the code that's common to both platforms simply imports the module/component while being oblivious to which platform it's actually running on. We used this technique extensively at FB. It'd be nice if rnws could support this too.

My first question is whether this would be better done using the `exclude` config option on the `loaders` in `webpack.config.js`, or using the `ignore` option for `babel-loader`? And the second is the same as yours: could we support two loader configurations in the same `webpack.config.js`?